applications/app/views/fragments/gallerySlot.scala.html

```diff
@@ -1,8 +1,16 @@
 @(
     id: Int,
     isMobile: Boolean = false
-)
+)(implicit request: RequestHeader)
 @import conf.switches.Switches.CommercialSwitch
+@import mvt.CommercialGalleryBannerAds
+
+@additionalSizes = @{
+    if(CommercialGalleryBannerAds.isParticipating)
+        Seq("970,250", "970,350")
+    else
+        Seq.empty
+}
```

Comment:
Hey Thomas, good point...but I think I'll leave it for now as, once this test is complete (in a week), this logic will be removed and the sizes will just be added to the hard-coded string.